OasisDEX/oasis-borrow
# consume low-tier on close version of lib

## Changes 👷‍♀️

- consumes a new version of the library displays the low tier version on close
- related to https://github.com/OasisDEX/oasis-earn-sc/pull/205
  
## How to test 🧪

1. open earn v3 position
2. go to close: oasis fee should be **0.07%** of the WSTETH to swap
3. open earn v2 position
4. go to close: oasis fee should be **0.2%** of the WSTETH to swap
